src/App.tsx

```diff
@@ -1,24 +1,48 @@
-import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import './SectionView.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFaceSmile } from '@fortawesome/free-solid-svg-icons';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="parent">
+      <div className="section-title">
+        <div className='text'>
+          <h1> Intro </h1>
+          <p>This intro section consists of a tuba quartet in the style of DJ Templeton & The Windsurfers</p>
+        </div>
+        <div className='buttons'>
+          <button> 42 revisions </button>
+          <button> save revision </button>
+        </div>
+      </div>
```

Comment:
Would be cool to extract data out of this, and possibly give it its own component

```ts
const sectionData = {
    name: 'Intro',
    description: 'This intro section consists of a tuba quartet in the style of DJ Templeton & The Windsurfers',
    numRevisions: 42,
};
```

Then use it like:

```suggestion
        <div className='text'>
          <h1>{sectionData.name}</h1>
          <p>{sectionData.description}</p>
        </div>
        <div className='buttons'>
          <button> {sectionData.numRevisions} revisions </button>
          <button> save revision </button>
        </div>
      </div>
```

Reply:
Same with other user-provided data in the UI

Reply:
Getting the code to be more "data friendly" rather than having the data itself hardcoded into the JSX is my main "request" for the PR review at the moment